### DISCUSSION TITLE
openDTU verbindet sich nach WLAN-Abbruch nicht wieder

### DISCUSSION THREAD

**Discussion by RaimundSch** [tbnobody/OpenDTU#1264] [Q&A]
Hallo,
erst einmal an alle die an diesem Projekt mitgearbeitet haben ein Dankeschön.

In der letzten Zeit habe ich das Problem, dass sich mein Router aufhängt und sich dann von allein wieder hochfährt (Klärung mit Telekom in Arbeit). 
Dabei verbinden sich nach Routerneustart alle anderen ESP (DalyToMQTT, VictronToMQTT, Heizungssteuerung usw. ) wieder mit dem WLAN nur der ESP32 mit openDTU nicht. Auch nicht nach 2 - 3 Stunden. Die openDTU ist dann per Browser nicht erreichbar und wird im Router auch nicht als verbunden gelistet.
Trenne ich den ESP32 von der Stromversorgung und schließe ihn wieder an ist in wenigen Sekunden die Verbindung hergestellt.

Ich habe einfache Kenntnisse in der C-Programmierung aber in diesem umfangreichen Projekt traue ich mich nicht ohne Hilfe im Sourcecode herumzuwerkeln.

Weiß jemand von Euch wie ich die openDTU dazu bringe sich wieder mit dem WLAN zu Verbinden.

Grüße

**Comment by brekemeier** [comment]
Ich habe die neue Version 23.8.22 aktualisiert und meinen Router neu gestartet,  Der AP wird nicht erzeugt. Nach ca. 2-4 Min. war openDTU wieder verfügbar und ich habe meine Geräte in den DHCP-Lease (Router) eingetragen, sodass sie ihre IP beibehalten.
Gruß zurück
